motto/agents/data_processors/doc_processor.py

```diff
@@ -0,0 +1,84 @@
+import markdown
+import tiktoken
+from bs4 import BeautifulSoup
+import nltk
+
+CHUNK_SIZE = 200
+MIN_CHUNK_SIZE_CHARS = 350  # The minimum size of each text chunk in characters
+MIN_CHUNK_LENGTH_TO_EMBED = 5  # Discard chunks shorter than this
+MAX_NUM_CHUNKS = 300  # The maximum number of chunks to generate from a text
+ENDPAGE_NUM = 282  # the last page number
+
+
+class DocProcessor:
+    encoding = tiktoken.encoding_for_model("gpt-3.5-turbo")
+
+    @staticmethod
+    def clear_text(file: str) -> str:
+        """Saves the contents of the given markdown file to a string.
+
+        Args:
+            link (str): Path to the markdown file.
+
+        Returns:
+            str: Cleaned markdown content.
+        """
+        with open(file, "r", encoding="utf-8") as input_file:
+            text = input_file.read()
+
+        html = markdown.markdown(text)
```

Comment:
So this class will work only for markdown files. It should probably be reflected somewhere  (Maybe in the name of the class, or at least in the comments ).

Reply:
> So this class will work only for markdown files. It should probably be reflected somewhere (Maybe in the name of the class, or at least in the comments ).

It works for txt file too, I guess It will work for any text file